AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreConfigurationTests.swift

```diff
@@ -0,0 +1,150 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//
+//  DataStoreConfigurationTests.swift
+//  AWSDataStoreCategoryPluginIntegrationTests
+//
+//  Created by Guo, Rui on 6/15/20.
+//  Copyright © 2020 Amazon Web Services. All rights reserved.
+//
+
+import XCTest
+
+import AmplifyPlugins
+import AWSMobileClient
+import AWSPluginsCore
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class DataStoreConfigurationTests: XCTestCase {
+
+    func testConfigWithSameSchema() throws {
+
+        let databaseCreationDone = expectation(description: "Old database deleted and new database recreated")
```

Comment:
across different test runs, the database is not created on the first save after configuration. let's just keep it what you are using it for, like `saveSuccess = expectation(description: "Save was successful")` then if you want to explain the overall test, can just add context in a test comment